code/src/components/Album.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './album.css';
+import { Artist } from './Artist'
+import { Icon } from './Icon'
+
+// Körs en gång för varje varv i map, returnerar bild, artist coh album
+export const Album = props => {
+    return (
+        <section className="wrapper">
+            <article className="album">
+                <div className="image-container">
+                    <img src={props.album.images[1].url} className="cover-img" alt="" />
```

Comment:
Don't leave the `alt`-tag empty! You should always include an `alt`-tag of course but if you for some reason don't want to - leave out the whole attribute, don't give it an empty string as value. Also, for better readability - attributes on new lines:
```
<img
  src={props.album.images[1].url}
  className="cover-img"
  alt={props.album.name}
/>
```

Reply:
Fixed